exercises/java-primitives/src/main/java/com/cbfacademy/StringExercises.java

```diff
@@ -3,19 +3,68 @@
 public class StringExercises {
 
     public String fromCharacters(char[] characters) {
-        // TODO - Write code that creates a String from the input array of characters
-        throw new RuntimeException("Not implemented");
+        //TODO - Write code that creates a String from the input array of characters
+
+        String sentence = new String (characters);
+        return sentence;
```

Comment:
As before - no need to create a variable and then return it on the next line. Could just be a return statement